assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -511,7 +511,11 @@
 											<span>$service.getGradeForSubmitter($submission, $user.getId())#if ($assignment.TypeOfGrade.ordinal() == 3)/$service.getMaxPointGradeDisplay($assignment.ScaleFactor, $assignment.MaxGradePoint)#end</span>
 										#end
 									#else
-										<span>-#if ($assignment.TypeOfGrade.ordinal() == 3)/$service.getMaxPointGradeDisplay($assignment.ScaleFactor, $assignment.MaxGradePoint)#end</span>
+										#if ($service.getGradeForSubmitter($submission, $user.getId()).equals(""))
+											<span>-#if ($assignment.TypeOfGrade.ordinal() == 3)/$service.getMaxPointGradeDisplay($assignment.ScaleFactor, $assignment.MaxGradePoint)#end</span>
+										#else
+											<span>$service.getGradeForSubmitter($submission, $user.getId())#if ($assignment.TypeOfGrade.ordinal() == 3)/$service.getMaxPointGradeDisplay($assignment.ScaleFactor, $assignment.MaxGradePoint)#end</span>
```

Comment:
As far as I see, it's the same line as above right?

https://github.com/sakaiproject/sakai/pull/11462/files#diff-fb5a37681fb8d992097df7789ee9d1e951038a7ebbb7535400827251b2274921R511